src/proto/Aeternity.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package TW.Aeternity.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    string from_address = 1;
+    string to_address = 2;
+    uint64 amount = 3;
+    uint64 fee = 4;
+    string payload = 5; //optional
+    uint64 ttl = 6;
```

Comment:
can you add comments for each field?

Reply:
why do you think each field needs comment, I think they are self explanatory. 
May payload should also be documented..

Reply:
I added comments to addresses, payload and ttl, the rest is redundant imho